### PR TITLE
feat: Webassembly setup

### PR DESCRIPTION
**Pull request by haakonflatval-cognite**
# Description

This is a suggestion on how the Webassembly setup in Reveal could look.

It includes a small function call from `parseEpt` for the sole purpose of demonstration. Feedback is welcome!

# Checklist:

Here is a checklist that should completed before merging this given feature. 
Any shortcomings from the items below should be explained and detailed within the contents of this PR.

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [x] I have commented my code in hard-to-understand areas.
- [x] I have made corresponding changes to the public documentation.
- [x] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [x] I have refactored the code for readability to the best of my ability.
- [x] I have checked that my changes do not introduce regressions in the public documentation.
- [x] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [x] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [x] I have added TSDoc to any public facing changes.
- [x] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [x] I have noted down and am currently tracking any technical debt introduced in this PR.
- [x] I have thoroughly thought about the architecture of this implementation.
- [x] I have asked peers to test this feature.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -42,6 +42,9 @@ jobs:
     #   working-directory: viewer
     #   run: yarn run build:prod
 
+    - name: Override Rust version
+      run: rustup override set nightly
```
Comment (christjt) [inline .github/workflows/ci.yml:46]: No nightly please :)

File: viewer/wasm/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+$ rustup override set nightly
+```
+
+in this directory.
+
+To build the crate for the web, we use `wasm-pack`:
+
+```
+$ wasm-pack build --target web
+```
```
Comment (christjt) [inline viewer/wasm/README.md:19]: I guess you also need to install rust to build this? Some notes in the /viewer readme about this setup should probably be there.

**Review by christjt — CHANGES_REQUESTED** [review]
Overall looks good.

I think we should try to set this up such that we can have .wasm as part of a package. What happens if we want to add some more .wasm things (other than point clouds), then should all of this live in /wasm folder? Having a rust crate per package (that needs it) makes more sense to me, not sure what side effects that has though. 

Current setup is kinda like creating a /typescript folder and putting everything in there.

File: viewer/wasm/README.md (hunk 2 above)
Comment (haakonflatval-cognite) [reply viewer/wasm/README.md]: I've removed this README and added more information to the one in `/viewer` instead.

**Review by christjt — APPROVED** [review]
LGTM, other than a tiny error in .gitignore file.

Also please double check that building all v8 related stuff (i3d/f3d/ctm etc.) still works and that nothing went wrong in the moving of files.

**Pull request merged**